### PR TITLE
Containerize Flask App

### LINKED ISSUE #11: Containerize Flask App

### PR DESCRIPTION
**Pull request by jemisonf**
Fix #11

### CODE REVIEW

**Review by ianbrown9475 — APPROVED** [review]
Looks good. I added a readme with Docker instructions. Feel free to merge when ready.

File: api/readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+1. Build the Docker image
+
+```shell
+$ docker build -t msgr .
+```
+
+2. Run the Docker container
+
+```shell
+$ docker run --name msgr -d msgr
```
Comment (jemisonf) [inline api/readme.md]: Does this run with exposed ports by default?

**Review by jemisonf — COMMENTED** [review]
Thanks for adding the readme, one question though

File: api/readme.md (hunk 1 above)
Comment (ianbrown9475) [reply api/readme.md]: Right, good catch. Should be fixed now.

**Pull request merged**